conf/defaults.ini

```diff
@@ -260,6 +244,21 @@ external_manage_info =
 viewers_can_edit = false
 
 [auth]
+# Login cookie name
+login_cookie_name = grafana_session
+
+# The lifetime (days) an authenticated user can be inactive before being required to login at next visit. Default is 7 days.
+login_maximum_inactive_lifetime_days = 7
+
+# The maximum lifetime (days) an authenticated user can be logged in since login time before being required to login. Default is 30 days.
```

Comment:
have we decided on this?

Forcing people to login every X breaks tv screens.

Reply:
I'm going to create a separate issue for this

Reply:
#15287